docroot/modules/custom/layout_builder_custom/src/Plugin/Display/ListBlock.php

```diff
@@ -41,6 +41,8 @@ public function optionsSummary(&$categories, &$options) {
       'configure_filters' => $this->t('Customize filters in block'),
       // Add use_more option summary.
       'use_more' => $this->t('Display more link'),
+      // Add general_help_text field.
+      'general_help_text' => $this->t('General help text'),
```

Comment:
I think this should be an empty string since the default value could slip through somewhere.